### DISCUSSION TITLE
Trouble mapping mvn to strictly-maven

### DISCUSSION THREAD

**Discussion by onnimikki** [jgneff/strictly-maven#1] [General]
Hi,

it seems that making the alias between mvn and strictly-maven isn't working on my Raspberry Pi.  Not sure what I've done wrong.

Here is the output of 

`snap info strictly-maven
`
and then

`type mvn
`

but the alias doesn't appear to work:

drsmith@raspberrypi:~ $ mvn
bash: strictly-maven: command not found

...

drsmith@raspberrypi:~ $ snap info strictly-maven
name:      strictly-maven
summary:   Apache Maven™ in a strictly-confined snap
publisher: John Neffenger (jgneff)
store-url: https://snapcraft.io/strictly-maven
contact:   https://github.com/jgneff/strictly-maven/discussions
license:   Apache-2.0
description: |
  Apache Maven is a build automation tool used primarily for Java
  projects. Maven can also be used to build and manage projects
  written in C#, Ruby, Scala, and other languages. Maven projects
  are configured by a file with a Project Object Model (pom.xml).

  This package provides the latest release of Maven built directly
  from its source code on GitHub. If the OpenJDK Snap package is
  also installed, this package connects to it automatically for the
  location of its Java Development Kit.

  This Snap package is strictly confined, running in complete isolation
  with only limited access to your system. For important information
  about running Maven in a confined environment, see the README file
  for this package on GitHub:

  https://github.com/jgneff/strictly-maven

  This is a convenience binary of the Maven tool and not a release
  by the Apache Software Foundation. For official releases, see:

  https://maven.apache.org

  Apache Maven and the Maven logo are either registered trademarks or
  trademarks of the Apache Software Foundation in the United States
  and/or other countries.
commands:
  - strictly-maven
snap-id:      D5wwlOFMIh7UVGbHatOV61Gjh6MxMWxN
tracking:     latest/stable
refresh-date: today at 15:13 BST
channels:
  latest/stable:    3.9.2 2023-05-11 (10) 9MB -
  latest/candidate: ↑
  latest/beta:      ↑
  latest/edge:      ↑
installed:          3.9.2            (10) 9MB -
drsmith@raspberrypi:~ $ type mvn
mvn is aliased to `strictly-maven'

**Comment by onnimikki** [comment]
Ah, it appears that the /snap/bin folder is not recognized in my path.  Need to fix that.

James


> Hi,
> 
> it seems that making the alias between mvn and strictly-maven isn't working on my Raspberry Pi. Not sure what I've done wrong.
> 
> Here is the output of
> 
> `snap info strictly-maven ` and then
> 
> `type mvn `
> 
> but the alias doesn't appear to work:
> 
> drsmith@raspberrypi:~ $ mvn bash: strictly-maven: command not found
> 
> ...
> 
> drsmith@raspberrypi:~ $ snap info strictly-maven name: strictly-maven summary: Apache Maven™ in a strictly-confined snap publisher: John Neffenger (jgneff) store-url: https://snapcraft.io/strictly-maven contact: https://github.com/jgneff/strictly-maven/discussions license: Apache-2.0 description: | Apache Maven is a build automation tool used primarily for Java projects. Maven can also be used to build and manage projects written in C#, Ruby, Scala, and other languages. Maven projects are configured by a file with a Project Object Model (pom.xml).
> 
> This package provides the latest release of Maven built directly from its source code on GitHub. If the OpenJDK Snap package is also installed, this package connects to it automatically for the location of its Java Development Kit.
> 
> This Snap package is strictly confined, running in complete isolation with only limited access to your system. For important information about running Maven in a confined environment, see the README file for this package on GitHub:
> 
> https://github.com/jgneff/strictly-maven
> 
> This is a convenience binary of the Maven tool and not a release by the Apache Software Foundation. For official releases, see:
> 
> https://maven.apache.org
> 
> Apache Maven and the Maven logo are either registered trademarks or trademarks of the Apache Software Foundation in the United States and/or other countries. commands:
> 
> * strictly-maven
>   snap-id:      D5wwlOFMIh7UVGbHatOV61Gjh6MxMWxN
>   tracking:     latest/stable
>   refresh-date: today at 15:13 BST
>   channels:
>   latest/stable:    3.9.2 2023-05-11 (10) 9MB -
>   latest/candidate: ↑
>   latest/beta:      ↑
>   latest/edge:      ↑
>   installed:          3.9.2            (10) 9MB -
>   drsmith@raspberrypi:~ $ type mvn
>   mvn is aliased to `strictly-maven'

**Comment by jgneff** [comment]
> Ah, it appears that the /snap/bin folder is not recognized in my path. Need to fix that.

Do you have this working now, James? Sorry for my delay in responding. I was away on vacation for the past few weeks.
